services/metric/registry_test.go

```diff
@@ -158,6 +158,7 @@ func benchmarkRegistryGetCounterAndInc(b *testing.B, concurrency int) {
 	end.Add(concurrency)
 	for i := 0; i < concurrency; i++ {
 		go func() {
+			start.Wait()
```

Comment:
[question] why we are doing this  ? 🤔

Reply:
so that nobody starts before the benchmark's timer starts, but not 100% sure it is needed